docs/getting_started/configuration/poly_new/guide.rst

```diff
@@ -445,25 +449,47 @@ With the final configuration:
 
 Launch ERT, choose the "Ensemble Smoother" and hit "Run Experiment".
 
-As simulations run, you'll notice the updated parameters yield results that better align with observations.
+Plot prior and posterior ensembles and notice that the updated parameters yield responses that better align with observations.
 
-In the "Plotting window", the ``POLY_RES`` plot will now display a yellow background, denoting attached observations
+In the "Plotting window", the ``POLY_RES`` plot will now display a yellow background, denoting that observations are available.
 Black dots and lines represent observed values and uncertainties, respectively.
-Different iterations can be viewed by selecting "default" and "default_smoother_update."
+Ensembles can be selected / deselected in the "Plot ensemble" section.
 
 .. image:: with_observations/plot_obs.png
 
 Evaluating the Updated Parameters
 *********************************
-Examine the improved estimates for `a`, `b`, and `c`.
-Though not perfect, they're better than the initial guesses. ERT's updating algorithms have reduced uncertainty:
+Examine the improved estimates for ``a``, ``b``, and ``c``.
+Though not perfect, they're better than the initial guesses.
 
 .. image:: with_observations/coeff_a.png
 
 .. image:: with_observations/coeff_b.png
 
 .. image:: with_observations/coeff_c.png
 
+Adding Observations
+*******************
+Notice that there is not much improvement in the estimate of parameter ``b``.
+Let's try adding more obeservations with lower uncertainty and run a new experiment.
+
+The following code generates synthetic observations as before, but now we generate 50 instead of 5, and reduce their uncertainty from ``0.2*p(x)`` to ``0.1*p(x)``.
+Run this script and copy results to ``poly_obs_data.txt``.
+
+.. literalinclude:: with_more_observations/generate_synthetic_observations.py
+
+Modify ``poly_eval.py`` to generate 50 responses:
+
+.. literalinclude:: with_more_observations/poly_eval.py
+
+Remove index list from ``observations`` as we now use all 50 observations:
+
+.. literalinclude:: with_more_observations/observations
+
+Re-run ``Ensemble smoother`` and notice that the estimate of ``b`` has improved:
```

Comment:
Perhaps add a bit more explaining why that improves the result?

Reply:
I'll add more when inspiration strikes.